cloud/cloud.go

```diff
@@ -0,0 +1,10 @@
+// Package cloud contains app-related functionality.
+package cloud
+
+// Metadata contains app-related information about the robot.
+type Metadata struct {
+	PrimaryOrgID  string
+	LocationID    string
+	MachineID     string
+	MachinePartID string
+}
```

Comment:
note that I am replacing `RobotPartID` with `MachinePartID` instead of keeping both. this is somewhat breaking, but given that we are only exposing this struct to external users now I think it's okay. will make an announcement either way.

Reply:
it's a slow transition, but long-term we want to use `Machine` instead of `Robot`. no rush to change it for existing code, but this is certainly the preference for new code AFAIK